graal/backends/core/colic.py

```diff
@@ -101,13 +105,15 @@ def fetch(self, category=CATEGORY_COLIC_NOMOS, paths=None,
     def metadata_category(item):
         """Extracts the category from a Code item.
 
-        This backend generates two types of item which can be:
-        'code_license_nomos' or 'code_license_scancode'.
+        This backend generates the following types of item:
+        'code_license_nomos', 'code_license_scancode' or 'code_license_scancode_cli.
```

Comment:
code_license_scancode_cli misses a `'` at the end. The line seems to long, maybe it could be rewritten as:
```
This backend generates the following types of item:
- 'code_license_nomos'
- 'code_license_scancode'
- 'code_license_scancode_cli'
```
What do you think ?